app/templates/components/modals/add-user-role-modal.hbs

```diff
@@ -0,0 +1,48 @@
+<div class="header">
+  {{t 'Add Role'}}
+</div>
+<div class="content">
+  <form class="ui form" id="add-user-role-form" autocomplete="off" {{action 'addRole' on='submit' preventDefault=true}}>
+    <div class="field">
+      <label class="required">
+        {{t 'User Email'}}
+      </label>
+      {{input type='text' name='user_email'}}
+    </div>
+    <div class="field">
+      <label class="required">
+        {{t 'User role'}}
+      </label>
+      {{#ui-dropdown class='fluid selection' required=true}}
+        {{input type='hidden' name='user_role' required=true}}
+        <i class="dropdown icon"></i>
+        <div class="default text">
+          {{t 'Select a role'}}
+        </div>
+        <div class="menu">
+          <div class="item" data-value="track_organizer">
+            {{t 'Track Organizer'}}
+          </div>
+          <div class="item" data-value="coorganizer">
+            {{t 'Co-organizer'}}
+          </div>
+          <div class="item" data-value="moderator">
+            {{t 'Moderator'}}
+          </div>
+          <div class="item" data-value="registrar">
+            {{t 'Registrar'}}
+          </div>
+        </div>
+      {{/ui-dropdown}}
+    </div>
+  </form>
+</div>
+<div class="actions">
+  <button type="button" class="ui black button" {{action 'close'}}>
```

Comment:
I don't think you need a close action here. Add a class `cancel` to the button and semantic ui will take care of it for you.